test/unit_tests/modules/util/type-checkers-spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var typeCheckers = require(
+  '../../../../src/static/js/modules/util/type-checkers.js'
+);
+
+var undefinedVar;
+var blankVar = '';
+var aString = 'bar';
+var aNum = 42;
+var aDate = new Date( 2011, 7, 21 );
+function aFunction() {
+  return true;
+}
+var anObject = {
+  a: '1',
+  b: '2',
+  c: '3'
+};
+var anArray = [ 1, 2, 3 ];
+
+describe( 'TypeCheckers isUndefined', function() {
+  it( 'should identify undefined variables', function() {
+    expect( typeCheckers.isUndefined( undefinedVar ) ).to.be.true;
```

Comment:
Is there a way we could have consistent assertions between the unit tests and the browser acceptance tests? E.g. both have `to.be.true` or `toBe( true )` syntax? Preferably the latter so the ESLint `dot-notation` ignore isn't necessary. 
Not something that needs to hold up this PR, but just wondering how we might make a unified test syntax.